.github/workflows/collector.yml

```diff
@@ -29,12 +34,13 @@ env:
 
 jobs:
   build-collector-image:
-    name: Build Collector
-    runs-on: ubuntu-24.04
+    name: Local collector image ${{ matrix.arch }}
     strategy:
-      fail-fast: false
       matrix:
-        arch: [amd64, ppc64le, arm64]
+        arch: [amd64, arm64, ppc64le]
+        exclude:
+          - arch: ${{ contains(inputs.architectures, 'ppc64le') && '' || 'ppc64le' }}
+    runs-on: ${{ (matrix.arch == 'arm64' && 'ubuntu-22.04-arm') || 'ubuntu-22.04' }}
```

Comment:
Any reason in particular why we rolled back to ubuntu-22.04? Or is it just copy-pasting from the Falco workflow?

If there's no strong reason, I would suggest sticking to 24.04, just to be on latest.